client/landing/gutenboarding/components/login-form/index.tsx

```diff
@@ -15,14 +15,8 @@ import EnterUsernameOrEmailForm from './enter-username-or-email';
 import EnterPasswordForm from './enter-password';
 import './style.scss';
 
-// TODO: deploy this change to @types/wordpress__element
-declare module '@wordpress/element' {
-	// eslint-disable-next-line no-shadow
-	export function createInterpolateElement(
-		interpolatedString: string,
-		conversionMap: Record< string, ReactElement >
-	): ReactNode;
-}
+// TODO: remove after @types/wordpress__element update
+/*#__PURE__*/ import '../../gutenberg-types-patch';
```

Comment:
I believe we should do this 1 time and have the type available throughout. Does it work if we do it in `client/gutenboarding/landing/index` and not everywhere we need these types?